capstone-app/src/app/modules/maps/map/map.component.ts

```diff
@@ -27,47 +42,122 @@ export class MapComponent {
     setTimeout(() => {
       this.locationSearch();
     });
+    this.nearSearch = new google.maps.places.PlacesService(this.mapComponent._googleMap);
```

Comment:
This is in ngAfterViewInit. Won't it cause there to be a 'ExpressionChangedAfterItHasBeenCheckedError' Thrown. Wouldn't it be more appropriate in ngAfterContentChecked()?

Reply:
I probably need Matt/Nicholas to comment about this

Reply:
no it won't i put there because we only want this called once and if we put in the setTimeout() it will call multiple times which is something we don't want

Reply:
Nope. Since it's just defining the service and not affecting the dom at this point, it won't cause any errors.